tests/trainer/test_trainer.py

```diff
@@ -931,6 +971,46 @@ def training_step_and_backward(split_batch, batch_idx, opt_idx, optimizer, hidde
     trainer.fit(model)
 
 
+@RunIf(min_gpus=1, amp_native=True)
+def test_gradient_clipping_by_value_fp16(tmpdir):
+    """
+    Test gradient clipping by value with fp16
+    """
+    tutils.reset_seed()
+
+    model = EvalModelTemplate()
```

Comment:
please use the `BoringModel` or one of its variants instead of the `EvalModelTemplate`